### PR TITLE
[0.74] Promote 74 to latest

### PR DESCRIPTION
**Pull request by TatianaKapos**
## Description
Promote 74 to latest
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/react-native-windows/pull/12986)

### CODE REVIEW

**Review by marlenecota — APPROVED** [review]

File: packages/@office-iss/react-native-win32/package.json [hunk 1]
```diff
@@ -95,11 +95,11 @@
     "react-native": "0.74.0-rc.9"
   },
   "beachball": {
-    "defaultNpmTag": "preview",
+    "defaultNpmTag": "latest",
     "disallowedChangeTypes": [
       "major",
       "minor",
-      "patch"
+      "prerelease"
```
Comment (jonthysell) [inline packages/@office-iss/react-native-win32/package.json:102]: I think we want to add these to all the packages too (for future proofing)?

```suggestion
      "prerelease",
      "premajor",
      "preminor",
      "prepatch"
```

File: packages/@react-native-windows/cli/package.json [hunk 2]
```diff
@@ -74,11 +74,11 @@
     "src/powershell"
   ],
   "beachball": {
-    "defaultNpmTag": "preview",
+    "defaultNpmTag": "latest",
     "disallowedChangeTypes": [
       "major",
       "minor",
-      "patch"
+      "prerelease"
```
Comment (jonthysell) [inline packages/@react-native-windows/cli/package.json:81]:
```suggestion
      "prerelease",
      "premajor",
      "preminor",
      "prepatch"
```

File: packages/@react-native-windows/codegen/package.json [hunk 3]
```diff
@@ -57,11 +57,11 @@
     "src"
   ],
   "beachball": {
-    "defaultNpmTag": "preview",
+    "defaultNpmTag": "latest",
     "disallowedChangeTypes": [
       "major",
       "minor",
-      "patch"
+      "prerelease"
```
Comment (jonthysell) [inline packages/@react-native-windows/codegen/package.json:64]:
```suggestion
      "prerelease",
      "premajor",
      "preminor",
      "prepatch"
```

File: packages/@react-native-windows/find-repo-root/package.json [hunk 4]
```diff
@@ -32,11 +32,11 @@
     "typescript": "5.0.4"
   },
   "beachball": {
-    "defaultNpmTag": "preview",
+    "defaultNpmTag": "latest",
     "disallowedChangeTypes": [
       "major",
       "minor",
-      "patch"
+      "prerelease"
```
Comment (jonthysell) [inline packages/@react-native-windows/find-repo-root/package.json:39]:
```suggestion
      "prerelease",
      "premajor",
      "preminor",
      "prepatch"
```

File: packages/@react-native-windows/fs/package.json [hunk 5]
```diff
@@ -36,11 +36,11 @@
     "lib-commonjs"
   ],
   "beachball": {
-    "defaultNpmTag": "preview",
+    "defaultNpmTag": "latest",
     "disallowedChangeTypes": [
       "major",
       "minor",
-      "patch"
+      "prerelease"
```
Comment (jonthysell) [inline packages/@react-native-windows/fs/package.json:43]:
```suggestion
      "prerelease",
      "premajor",
      "preminor",
      "prepatch"
```

File: packages/@react-native-windows/package-utils/package.json [hunk 6]
```diff
@@ -34,11 +34,11 @@
     "typescript": "5.0.4"
   },
   "beachball": {
-    "defaultNpmTag": "preview",
+    "defaultNpmTag": "latest",
     "disallowedChangeTypes": [
       "major",
       "minor",
-      "patch"
+      "prerelease"
```
Comment (jonthysell) [inline packages/@react-native-windows/package-utils/package.json:41]:
```suggestion
      "prerelease",
      "premajor",
      "preminor",
      "prepatch"
```

File: packages/@react-native-windows/telemetry/package.json [hunk 7]
```diff
@@ -51,11 +51,11 @@
     "lib-commonjs"
   ],
   "beachball": {
-    "defaultNpmTag": "preview",
+    "defaultNpmTag": "latest",
     "disallowedChangeTypes": [
       "major",
       "minor",
-      "patch"
+      "prerelease"
```
Comment (jonthysell) [inline packages/@react-native-windows/telemetry/package.json:58]:
```suggestion
      "prerelease",
      "premajor",
      "preminor",
      "prepatch"
```

File: vnext/package.json [hunk 8]
```diff
@@ -92,11 +92,11 @@
     "react-native": "0.74.0-rc.9"
   },
   "beachball": {
-    "defaultNpmTag": "preview",
+    "defaultNpmTag": "latest",
     "disallowedChangeTypes": [
       "major",
       "minor",
-      "patch"
+      "prerelease"
```
Comment (jonthysell) [inline vnext/package.json:99]:
```suggestion
      "prerelease",
      "premajor",
      "preminor",
      "prepatch"
```

**Comment by TatianaKapos** [comment]
waiting till https://github.com/microsoft/react-native-windows/pull/12999#pullrequestreview-2025574765 gets merged in to push this :)

**Review by acoates-ms — APPROVED** [review]

**Pull request merged**